robotidy/cli.py

```diff
@@ -229,6 +230,12 @@ def print_transformers_list():
     help="Output diff of each processed file.",
     show_default=True,
 )
+@click.option(
+    "--color/--no-color",
```

Comment:
So, `--color` and `--no-color` are the names for the same option? This can be confusing when someone wants to disable coloring and will write `--color False`. The short version should be something like `--nc`.

Reply:
No, it's optional flag (without value): you can provide --color (which is default behaviour, like --overwrite) or you can provide opposite which is --no-color (same as --no-overwrite). 

robotidy --no-color
to disable
robotidy --color to retain default behaviour

and in configuration it's:
color = False